addons/viewport/src/shortcuts.ts

```diff
@@ -0,0 +1,59 @@
+import { API } from '@storybook/api';
+import { ADDON_ID } from './constants';
+import { MINIMAL_VIEWPORTS } from './defaults';
+
+const viewportsKeys = Object.keys(MINIMAL_VIEWPORTS);
```

Comment:
When a custom list of viewports is set, will it be registered?

Reply:
Oh good catch, I had not thought of that at all! Thank you for pointing that out, I'll be fixing it as soon as possible!